testing/trino-testing/src/main/java/io/trino/testing/AbstractTestDistributedQueries.java

```diff
@@ -1372,6 +1372,8 @@ private void testDataMapping(DataMappingTestSetup dataMappingTestSetup)
                 .add(new DataMappingTestSetup("double", "DOUBLE '1234567890123.123'", "DOUBLE '9999999999999.999'"))
                 .add(new DataMappingTestSetup("decimal(5,3)", "12.345", "99.999"))
                 .add(new DataMappingTestSetup("decimal(15,3)", "123456789012.345", "999999999999.99"))
+                .add(new DataMappingTestSetup("date", "DATE '0001-01-01'", "DATE '1582-10-04'")) // before julian->gregorian switch
```

Comment:
A test with negative dates would be good to have to catch issues in the datetimeformatters being used.

Reply:
Let me handle BC vs AD issue in #10231 so that we can make this PR small.